Can't load the plugin with typescript 3.2.2

When I try to load the plugin into angular7 project I get an Error
`ERROR in node_modules/web-streams-polyfill/dist/types/polyfill.d.ts(1,21): error TS2727: Cannot find lib definition for 'es2018.asynciterable'. Did you mean 'esnext.asynciterable'?`

I did some investigations and it seems the cause is typescript version I have. 3.2.2

NOTE it works as expected with typescript version 3.5.3 that I have in angular 8 app

I can't upgrade typescript version since it's required by angular 7. The only way to make your polyfill to work is to upgrade to angular 8.

It seems that in typescript 3.2  `asynciterable` is under esnext.asynciterable instead of es2018.asynciterable. 

Please advise. Thank you in advanced.

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -9,6 +9,10 @@
 > - 🏠 Internal
 > - 💅 Polish
 
+## Unreleased
+
+* 🐛 Fix type definitions to be compatible with TypeScript 3.3 and lower. ([#39](https://github.com/MattiasBuelens/web-streams-polyfill/issues/39), [#40](https://github.com/MattiasBuelens/web-streams-polyfill/pull/40))
+
 ## v2.0.5 (2019-10-08)
 
 * 👓 Align with [spec version `ae5e0cb`](https://github.com/whatwg/streams/tree/ae5e0cb41e9f72cdd97f3a6d47bc674c1f4049d1/) ([#33](https://github.com/MattiasBuelens/web-streams-polyfill/pull/33))

--- a/src/lib/readable-stream/async-iterator.ts
+++ b/src/lib/readable-stream/async-iterator.ts
@@ -1,4 +1,4 @@
-/// <reference lib="es2018.asynciterable" />
+/// <reference lib="esnext.asynciterable" />
 
 import { ReadableStream } from '../readable-stream';
 import {

--- a/src/target/es2018/stub/async-iterator-prototype.ts
+++ b/src/target/es2018/stub/async-iterator-prototype.ts
@@ -1,4 +1,4 @@
-/// <reference lib="es2018.asynciterable" />
+/// <reference lib="esnext.asynciterable" />
 
 export const AsyncIteratorPrototype: AsyncIterable<any> | undefined =
   Object.getPrototypeOf(Object.getPrototypeOf(async function* (): AsyncIterableIterator<any> {}).prototype);

--- a/src/target/es5/stub/async-iterator-prototype.ts
+++ b/src/target/es5/stub/async-iterator-prototype.ts
@@ -1,4 +1,4 @@
-/// <reference lib="es2018.asynciterable" />
+/// <reference lib="esnext.asynciterable" />
 
 export let AsyncIteratorPrototype: AsyncIterable<any> | undefined;
 

